test/gpg-asym/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+services:
+  backup:
+    image: offen/docker-volume-backup:${TEST_VERSION:-canary}
+    restart: always
+    environment:
+      BACKUP_CRON_EXPRESSION: 0 0 5 31 2 ?
+      BACKUP_FILENAME: test.tar.gz
+      BACKUP_LATEST_SYMLINK: test-latest.tar.gz.gpg
+      BACKUP_RETENTION_DAYS: ${BACKUP_RETENTION_DAYS:-7}
+      GPG_PUBLIC_KEY_RING: | 
+        -----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
I wonder if it would make sense to remove the duplication and instead pass the asc-file using GPG_PUBLIC_KEY_RING_FILE, which I would assume is what most people would be using in the wild as well. What do you think?

Reply:
What do you mean with remove the duplication. `GPG_PUBLIC_KEY_RING` and `-----BEGIN PGP PUBLIC KEY BLOCK`. Tbh, parsing a file would probably be cleaner, since putting the keyring in the compose.yaml makes the file unnecessarily large and hard to read.

Would you prefer to remove the `GPG_PUBLIC_KEY_RING` option altogether and introduce the `GPG_PUBLIC_KEY_RING_FILE` or a tandem solution. im fine with both but would prefer a standalone file solution.

Reply:
Sorry for not being clear. All configuration values can also be read from a file if you supply a location using `CONFIG_KEY_FILE`, so this already works out of the box, you can pass a full key using `GPG_PUBLIC_KEY_RING` or the location to a key file using `GPG_PUBLIC_KEY_RING_FILE`.

See the topmost note here https://offen.github.io/docker-volume-backup/reference/

Reply:
Nice. I will then pass the file in the compse.yaml

Reply:
done, [e44fd35](https://github.com/offen/docker-volume-backup/pull/456/commits/e44fd352e9d1383115cd917e47f8e32e39207219)